cmd/build-push-image/main.go

```diff
@@ -47,30 +46,55 @@ type options struct {
 	debug                 bool
 }
 
+var defaultOptions = options{
+	bitbucketAccessToken:  os.Getenv("BITBUCKET_ACCESS_TOKEN"),
+	bitbucketURL:          os.Getenv("BITBUCKET_URL"),
+	aquaUsername:          os.Getenv("AQUA_USERNAME"),
+	aquaPassword:          os.Getenv("AQUA_PASSWORD"),
+	aquaURL:               os.Getenv("AQUA_URL"),
+	aquaRegistry:          os.Getenv("AQUA_REGISTRY"),
+	imageStream:           "",
+	registry:              "image-registry.openshift-image-registry.svc:5000",
+	certDir:               "/etc/containers/certs.d",
+	imageNamespace:        "",
+	tlsVerify:             true,
+	storageDriver:         "vfs",
+	format:                "oci",
+	dockerfile:            "./Dockerfile",
+	contextDir:            "docker",
+	nexusURL:              os.Getenv("NEXUS_URL"),
+	nexusUsername:         os.Getenv("NEXUS_USERNAME"),
+	nexusPassword:         os.Getenv("NEXUS_PASSWORD"),
+	buildahBuildExtraArgs: "",
```

Comment:
For both buildah..ExtraArgs this was `"docker"` before, so this is the only 'real' change except for the refactor or?

Reply:
Correct. `docker` was a mistake (that should not have had any effect).